plugins/store/kics/main.go

```diff
@@ -92,15 +95,21 @@ func (s *Scanner) Start(config types.Config) {
 			return
 		}
 
-		rawOutputFile := filepath.Join(os.TempDir(), "kics.json")
+		// Ensure JSON format is always included,
+		// since it's the only format that can be consumed by VMClarity
+		clientConfig.ReportFormats = ensureJSONFormat(clientConfig.ReportFormats)
```

Comment:
we can drop `ensureJSONFormat` function and move this part directly to `createConfig` method

Reply:
I added it before returning the config, since if we get a config as input, and it does not have JSON included, it would be overwritten.

```go
func (s *Scanner) createConfig(scannerConfig *string) (*ScannerConfig, error) {
	config := ScannerConfig{
		PreviewLines:     3,
		// If we would place it here, it might be overwritten with the posted config
                // when we unmarshal on this.
		// ReportFormats:    []string{"json"},
		Platform:         []string{"Ansible", "CloudFormation", "Common", "Crossplane", "Dockerfile", "DockerCompose", "Knative", "Kubernetes", "OpenAPI", "Terraform", "AzureResourceManager", "GRPC", "GoogleDeploymentManager", "Buildah", "Pulumi", "ServerlessFW", "CICD"},
		MaxFileSizeFlag:  100,
		DisableSecrets:   true,
		QueryExecTimeout: 60,
		Silent:           true,
		Minimal:          true,
	}

	if scannerConfig == nil || *scannerConfig == "" {
		return &config, nil
	}

	if err := json.Unmarshal([]byte(*scannerConfig), &config); err != nil {
		return nil, fmt.Errorf("failed to unmarshal JSON config: %w", err)
	}

	// Ensure JSON format is always included,
	// since it's the only format that can be consumed by VMClarity
	config.ReportFormats = ensureJSONFormat(config.ReportFormats)

	return &config, nil
}
```